pkgs/by-name/te/teams-for-linux/package.nix

```
@@ -0,0 +1,126 @@
+{
+  lib,
+  stdenvNoCC,
+  buildNpmPackage,
+  fetchFromGitHub,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  nodejs,
+  electron_30,
+  alsa-utils,
+  which,
+  testers,
+  teams-for-linux,
+  nix-update-script,
+}:
+buildNpmPackage rec {
+  pname = "teams-for-linux";
+  version = "1.9.5";
+
+  src = fetchFromGitHub {
+    owner = "IsmaelMartinez";
+    repo = "teams-for-linux";
+    rev = "v${version}";
+    hash = "sha256-+rEGDg+/qvjCMhGHccb4p+CKOo/65RpkFT/WnCDlCgU=";
+  };
+
+  npmDepsHash = "sha256-vDRFFxkIQo5qU9gmkSwUhPz4FG2XbUNkTw6SCuvMqCc=";
+
+  nativeBuildInputs =
+    [
+      makeWrapper
+      nodejs
+    ]
+    ++ lib.optionals (stdenvNoCC.isLinux) [
+      copyDesktopItems
+    ];
+
+  env = {
+    # disable code signing on Darwin
+    CSC_IDENTITY_AUTO_DISCOVERY = lib.optionalString stdenvNoCC.isDarwin "false";
+    ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
+  };
+
+  buildPhase = ''
+    cp -r ${electron_30.dist} electron-dist
+    chmod -R u+w electron-dist
+
+    npm exec electron-builder -- \
+        --dir \
+        -c.npmRebuild=true \
+        -c.asarUnpack="**/*.node" \
+        -c.electronDist=electron-dist \
+        -c.electronVersion=${electron_30.version}
+  '';
+
+  installPhase = ''
+      runHook preInstall
```

```suggestion
    runHook preInstall
```
we use two spaces to format things in nix files